README.md

```diff
@@ -58,3 +58,14 @@ Logging based on the Serilog.AspNetCore NuGet package.
       </tr>
    </tbody>
 </table>
+
+## Migrations
+### Command to create a migration: 
+ `dotnet ef migrations add "name"`
```

Comment:
```suggestion
 `dotnet ef migrations add <Migration_Name>`
```
Um das einheitlich mit dem anderen `dotnet ef` Command darunter in der README zu haben